common/ts/publish-task.ts

```diff
@@ -35,7 +35,12 @@ export default async function publishTask(endpointType: EndpointType) {
     globalQualityGateStatus = 'ok';
   }
 
-  tl.debug(`Number of analyses in this build: ${taskReports.length}`);
+  if (!taskReports.length) {
+    tl.warning('No analyses found in this build! Please check your build configuration.');
```

Comment:
Hi, 

Can you run "npm run format" and "npm run lint" in you whole solution so that the CI will pass normally after ? 

Thanks.